### ISSUE TITLE
Error running python Deforum_Stable_Diffusion.py

### ISSUE THREAD

**Issue by daocha** [deforum-art/deforum-stable-diffusion#201]
I am encountering this error : 
```
/anaconda3/envs/dsd/lib/python3.10/site-packages/torchvision/image.so: undefined symbol: _ZN3c104cuda20CUDACachingAllocator9allocatorE'If you don't plan on using image functionality from `torchvision.io`, you can ignore this warning. Otherwise, there might be something wrong with your environment. Did you have `libjpeg` or `libpng` installed before building `torchvision` from source?
  warn(
Segmentation fault (core dumped)
```

Any idea? 
I tried to install : conda install jpeg libpng, and remove the dsd conda env, and then re-run `install_requirements.py`
But still got this error. 
I don't know how i can reinstall this `torchvision`.
I can not even uninstall `torchvision` using `conda uninstall torchvision`

**Comment by geekbleek** [comment]
I got the same issue - pip uninstalled torchvision and torch, and used conda to install them from the proper channel (pytorch and nvidia).  In my case, the pip repo wasn't installing pytorch built for cuda.  Then I had to do the same for xformers from the xformers channel. 

Was yours on a DGX or A100 system? I was trying on the NGC cuda base container image.  Will try and incorporate all my fixes (without having to make changes to the install_requirements.py file in the repo) into a Docker build file shortly.  Distributing the image wouldn't be easy as its massive w/o some serious enhancements (leg work) on the multi-stage build process.
